docs(numeric): Improve Parameters table (#1419)
telerik/blazor-docs: components/numerictextbox/overview.md

## Custom Format Strings

> The Blazor Numeric TextBox allows you to define your desired custom format through the `Format` parameter. Here are some examples:

Reviewer: ```suggestion
The Blazor Numeric TextBox allows you to define your desired custom format through the `Format` parameter, for example:
```

> >caption Using custom format strings with the Blazor NumericTextBox

Reviewer: Use the same component name throughout the entire document.
```suggestion
>caption Using custom format strings with the Blazor Numeric TextBox
```

| `Format` | `string` | The number format when the input is not focused. Read more at [Standard Numeric Format Strings in .NET](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings) |
| `Id` | `string` | The `id` attribute on the `<input />` element. |
| `InputMode` | `string` | The [`inputmode` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode) of the `<input />` element. |
| `Max` | `TValue`* <br /> (`T.MaxValue`) | The maximum value the input can accept. Must match the `Value` type. |
| `Min` | `TValue`* <br /> (`T.MinValue`) | The minimum value the input can accept. Must match the `Value` type. |
| `Placeholder` | `string` | The `placeholder` attribute of the HTML element. The placeholder will appear if the component is bound to a **nullable** value type and there is no value set. |

> | `Step` | `TValue`* <br /> (`1`) | The decimal value with which the value changes when using the arrows. Must maatch the `Value` type. |

Reviewer: ```suggestion
| `Step` | `TValue`* <br /> (`1`) | The decimal value with which the component value changes when using the arrows. Must match the `Value` type. |
```
```

> >tip To learn more about the appearance, anatomy, and accessibility of the NumericTextBox, visit the [Progress Design System documentation](https://www.telerik.com/design-system/docs/components/numerictextbox/)—an information portal offering rich component usage guidelines, descriptions of the available style variables, and globalization support details.

Reviewer: ```suggestion
>tip To learn more about the appearance, anatomy, and accessibility of the Numeric TextBox, visit the [Progress Design System documentation](https://www.telerik.com/design-system/docs/components/numerictextbox/)—an information portal offering rich component usage guidelines, descriptions of the available style variables, and globalization support details.
```

> ## NumericTextBox Reference and Methods

Reviewer: ```suggestion
## Numeric TextBox Reference and Methods
```

> The NumericTextBox has a `FocusAsync` method that enables programmatic focus. To use it, obtain reference to the component instance.

Reviewer: ```suggestion
The Numeric TextBox has a `FocusAsync` method that enables programmatic focus. To use it, obtain a reference to the component instance.
```
